public/api-merged.json

```diff
@@ -19843,28 +19843,6 @@
         "$ref": "#/definitions/Transformation"
       }
     },
-    "TrimDashboardCommand": {
```

Comment:
this change should come from #77249
@nikimanoledaki @yangkb09 maybe we need to investigate why the [swagger-gen CI step](https://drone.grafana.net/grafana/grafana/144430/8/3) didn't catch it